code/datums/supply_packs/weapons.dm

```diff
@@ -120,6 +120,16 @@
 	containername = "\improper XM88 Heavy Rifle crate"
 	group = "Weapons"
 
+/*
+/datum/supply_packs/gun/m707
+	name = "M707 Anti-Materiel Rifle crate (M707 x1)"
+	contains = list()
+	cost = 120
+	containertype = /obj/structure/closet/crate/secure/vulture
+	containername = "M707 crate"
+	group = "Weapons"
+*/
+
```

Comment:
assuming this is pending a decision on availability

Reply:
Decision's been made there; I'm leaving this in case such a decision gets reversed. Will add a comment